lib/components/util/types.ts

```diff
@@ -66,6 +65,17 @@ export interface Time {
   tripId: string
 }
 
+export interface PatternStopTimes {
```

Comment:
This type is different from the one returned in the GraphQL API, which is an issue for the Nearby View. Let's get this merged in and then we can figure out what to do in the Nearby View branch before merging it.